### ISSUE TITLE
`<Calendar/>` - Incomplete keyboard navigation

### ISSUE THREAD

**Issue by hepiyellow** [wix/wix-style-react#2677]
### Missing Tab order definition in Spec

The behavior we get from `react-day-picker` (which is the underlying implementation), is:
- Tab1 - Calendar - right/left arrows to navigate the month.
- Tab2 - PrevMonth Button (still right/left arrows navigate the month)
- Tab3 - NextMonth Button (still right/left arrows navigate the month)
- Tab4 - selectedDay - right/left arrows to navigate the days.

### Missing focus style for "whole-calendar-in-focus"

Currently when implementing `<CalendarPanel/>`, when the calendar gets focus we get a native outline (unless we disable it)

We can decide that if the component os part of a composition (e.g. `<DatePicker/>`) then the "whole-calendar" would not get focus (open a prop `tabIndex` that could be set to `-1`).
> I tried to add `tabIndex=-1` to `<DayPicker/>`, but then it doesn't get focus at all, instead of having the next/prev buttons get focus.

But we should have a focus style definition in the spec for standalone cases.

### Prev / Next Buttons

These don't get focus (no tabIndex)
We should be using the IconButton component, and then it would get focus and the appropriate focus style.

**Comment by shlomitc** [comment]
won't be fixed any time soon - will reopen on request
